### PR TITLE
SUS-5853 | make sandbox descriptor and job more generic

### PR DESCRIPTION
**Pull request by xkxd**
https://wikia-inc.atlassian.net/browse/SUS-5853

this also includes adding "sandbox_name" input in `sus-mediawiki-deployment-pipeline` job

### CODE REVIEW

File: docker/sandbox/Dockerfile-php.template [hunk 1]
```diff
@@ -1,8 +1,8 @@
 FROM artifactory.wikia-inc.com/sus/php-wikia-base:eb11f0c
 
-ENV WIKIA_DATACENTER="sjc"
-ENV WIKIA_ENVIRONMENT="sandbox"
-ENV HOSTNAME_OVERRIDE="sandbox-sus2"
+ENV WIKIA_DATACENTER="${SANDBOX_DATACENTER}"
+ENV WIKIA_ENVIRONMENT="${SANDBOX_ENVIRONMENT}"
```
Comment (macbre) [inline docker/sandbox/Dockerfile-php.template:4]: I think we can keep `sandbox` hardcoded here. We want to deploy sandbox here after all, right?

Comment (xkxd) [reply docker/sandbox/Dockerfile-php.template:4]: true, however, I've wanted to keep vars in sync with the ones from Jenkinsfile. "sandbox" is hardcoded there

**Review by macbre — APPROVED** [review]

**Comment by macbre** [comment]
Can we pass `datacenter` param value to `kubectl` command executed by this Jenkinsfile?

```
kubectl --context kube-sjc-prod
```

This way we will be able to deploy sandboxes to Reston.

**Comment by xkxd** [comment]
@macbre , sure we can, will do

**Review by pwojcik86 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**